spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -277,4 +277,32 @@ Package names that contain the types that are whitelisted for deserializing mess
 | `spring.cloud.gcp.pubsub.subscriber.parallel-pull-count` | The number of pull workers | No | The available number of processors
 | `spring.cloud.gcp.pubsub.subscriber.max-ack-duration-seconds` | The maximum period a message ack deadline will be extended in seconds | No | 3600
 | `spring.cloud.gcp.pubsub.subscriber.pull-endpoint` | The endpoint for synchronous pulling messages | No | pubsub.googleapis.com:443
+| `spring.cloud.gcp.pubsub.subscriber.retry.total-timeout-seconds`| pass-through setting of same name from `com.google.api.gax.retrying.RetrySetting` for subscriber |No |
```

Comment:
Is there some issue with actually copying in the descriptions of the properties rather than saying "pass-through setting"? I think it would be more user-friendly.

Reply:
The downside is that the description gets updated and also to reduce duplication of text. This also makes it clear to the user that he's directly touching that underlying setting on the client library and we aren't clouding the picture, which has its own value.

Reply:
I understand the hesitation to duplicate the documentation, but unless you can link directly to the description of this property in `com.google.api.gax.retrying.RetrySetting`, I would copy it in here and make the lives of our users easier, even if that makes our lives a bit harder. :smile:
The Pub/Sub library is GA now. So, we don't really expect these properties to change in the near future.

Reply:
OK it sounds like you want to copy the descriptions here. I can do that.

Reply:
Thanks!

Reply:
Done. please take a look.